byterun/caml/sys.h

```diff
@@ -18,6 +18,10 @@
 
 #include "misc.h"
 
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Why is this here when this change has consistently been rejected previously? (See for example [PR#2056](http://caml.inria.fr/mantis/view.php?id=2056).)